src/Components/ProjectTile.js

```diff
@@ -0,0 +1,103 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faChevronDown, faChevronUp } from '@fortawesome/free-solid-svg-icons';
+import { useHistory } from 'react-router';
+import Button from '@mui/material/Button';
+
+
+//
+//       Component: ProductTile
+//       Description: This component wraps the project in its separate box with an upvote and downvote.
+//
+//       Inputs:
+//           - NA
+//       Outputs:
+//          - NA
+const ProjectTile = ({ products, index, setProducts }) => {
+
+  const history = useHistory();
+  const upVote = () => {
+    const updatedProduct = { ...products[index] };
+    let currentVote = updatedProduct.upVoted ? 1 : (updatedProduct.downVoted ? -1 : 0);
+    updatedProduct.upVoted = !updatedProduct.upVoted;
+    updatedProduct.downVoted = false;
+    let newVote = updatedProduct.upVoted ? 1 : (updatedProduct.downVoted ? -1 : 0);
+    updatedProduct.votes = updatedProduct.votes - currentVote + newVote;
+    setProducts(products.map((product) => product.id === products[index].id ? updatedProduct : product));
+  };
+  const downVote = () => {
+    const updatedProduct = { ...products[index] };
+    let currentVote = updatedProduct.upVoted ? 1 : (updatedProduct.downVoted ? -1 : 0);
+    updatedProduct.downVoted = !updatedProduct.downVoted;
+    updatedProduct.upVoted = false;
+    let newVote = updatedProduct.upVoted ? 1 : (updatedProduct.downVoted ? -1 : 0);
+    updatedProduct.votes = updatedProduct.votes - currentVote + newVote;
+    setProducts(products.map((product) => product.id === products[index].id ? updatedProduct : product));
+  };
+  const goTo = (product) => () => {
+    history.push(`/${product}/getFeature`);
+  };
+  const capitalizeFirstLetter = (string) => {
+    return string.charAt(0).toUpperCase() + string.slice(1);
+  };
+
+  const deleteDiv = (event) => {
+    const product = event.target.parentNode.parentNode.parentNode;
+    product.parentNode.removeChild(product);
+
+  };
+  return (
+    <div className="child product">
+      <div className="product-container">
+        <div className="image-container">
+          <img src={products[index].image_url} alt={products[index].name} />
+        </div>
+        <div className="content">
+          <div className="product-content">
+            <span className="product-title" 
+            data-testid={"ptnav:"+index}
+            onClick={goTo(products[index].name)} 
+            style={{ marginTop: 'auto', marginBottom: 'auto' }}>
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/elric97/feature-hunt/pullRequest?prid=8494981)
```suggestion
            style={{ marginTop: "auto", marginBottom: 'auto' }}>
```